source/_integrations/switch.mqtt.markdown

```diff
@@ -33,6 +33,10 @@ availability:
   required: false
   type: list
   keys:
+    availability_template:
+      description: "Defines a [template](/docs/configuration/templating/#processing-incoming-data) to extract device's availability from the `topic`. To determine the devices's availability result of this template will be compared to `payload_available` and `payload_not_available`."
+      required: false
+      type: template
```

Comment:
This is not needed?

Reply:
A switch can become unavailable e.g. when a device is a mode that does not allow a switch to be set.